ethportal-api/src/types/cli.rs

```diff
@@ -170,6 +177,7 @@ impl Default for TrinConfig {
             bootnodes: Bootnodes::Default,
             external_addr: None,
             no_stun: false,
+            no_upnp: false,
```

Comment:
It seems like this means we're **enabling** upnp by default? Is that the desired behavior?

Reply:
The behavior is borrowed from geth's default behavior, where UPnP is used by default (and there is no flag to disable it).  Feel free to let me know if you have better thoughts.

Reply:
I believe enable by default is the desired behavior